src/js/providers/tracks-mixin.js

```diff
@@ -423,6 +425,10 @@ function disableTextTrack() {
     }
 }
 
+function getCurrentTextTrack() {
+    return this._textTracks && this._textTracks[this._currentTextTrackIndex];
```

Comment:
`this.getCurrentTextTrack()` should replace `this._textTracks[this._currentTextTrackIndex]` everywhere else in this file.

Reply:
great call